flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/ddl/AlterTableDropConstraintOperation.java

```diff
@@ -20,13 +20,15 @@
 
 import org.apache.flink.table.catalog.ObjectIdentifier;
 
-/** Operation of "ALTER TABLE ADD [CONSTRAINT constraintName] ..." clause. * */
+import org.apache.commons.lang3.StringUtils;
+
+/** Operation of "ALTER TABLE IF EXISTS ADD [CONSTRAINT constraintName] ..." clause. * */
 public class AlterTableDropConstraintOperation extends AlterTableOperation {
     private final String constraintName;
 
     public AlterTableDropConstraintOperation(
-            ObjectIdentifier tableIdentifier, String constraintName) {
-        super(tableIdentifier);
+            boolean ifExists, ObjectIdentifier tableIdentifier, String constraintName) {
```

Comment:
Maybe change this to `AlterTableDropConstraintOperation(ObjectIdentifier, String, boolean)`?